doc/source/index.rst

```diff
@@ -0,0 +1,29 @@
++++++++++++++++++++++++
+ rocPRIM Documentation 
++++++++++++++++++++++++
+
+.. toctree::
+   :hidden:
+
+   introduction
+   operations
+
+   devicemodule/index
+   blockmodule/index
+   warpmodule
+   threadmodule
+
+   iterators
+   intrinsics
+
+   glossary
+
+The ``rocPRIM`` is a header-only library providing HIP parallel primitives to ease the maintainability of performant and yet portable GPU-accelerated code on AMD ROCm platform.
+
+As it is the ROCM dual of Nvidia CUB ( https://github.com/NVIDIA/cub ), looking at the CUB documentation could be also interesting for you: https://nvlabs.github.io/cub/index.html
```

Comment:
Please remove this line. hipcub is our cub compatibility layer library.

Reply:
@v01dXYZ  I understand your concerns. The relation between the AMD libraries, rocPRIM and hipCUB, and the NVIDIA library CUB is not always clear. This is something that I want to be clear to end users. I don't have a clear plan on that yet and let me think about it.